docs/src/views/AccessibilityBehaviors.tsx

```diff
@@ -0,0 +1,253 @@
+            'aria-describedby': props['aria-describedby'],
+            'aria-disabled': !_.isNil(props['aria-disabled'])
+              ? props['aria-disabled']
+              : !!props['disabled']
+              ? true
+              : undefined,
+            [IS_FOCUSABLE_ATTRIBUTE]: !props['disabled'],
+            'aria-posinset': props.itemPosition,
+            'aria-setsize': props.itemsCount,
+          },
+        },
+        //...
+      })
+      `}
+    />
+
+    <Header as="h2" content="Overriding behaviors" />
+    <p>
+      User can override the default behavior by using the {code('accessibility')} attribute, as well
+      as override generated attributes:
+    </p>
+    <CodeSnippet label="App.jsx" value={`<Menu accessibility={tabListBehavior} />`} />
+    <Header as="h4" content="Creating new behavior" />
+    <p>
+      Beside of overriding the default behavior with an existent one, it is also possible to create
+      your own.
+    </p>
+    <CodeSnippet
+      label={'newBehavior.tsx'}
+      value={`
+      const newBehavior: Accessibility = (props: any) => ({
+        attributes: {
+          root: {
+            //...
+          },
+        },
+        keyActions: {
+          root: {
+            //...
+          },
+        },
+        focusZone: {},
+        focusTrap: {},
+        autoFocus: {},
+      })
+      `}
+    />
+
+    <p>
+      Also, if particular behavior matches your needs but you want to change some of the props, you
+      can create a new one based on existent behavior and override what is needed.
+    </p>
+    <p>
+      For example, using {code('menuBehavior')}, create a new one with overriden some of its
+      FocusZone's props.
+    </p>
+    <CodeSnippet
+      label="chatBehavior.ts"
+      value={`
+      const overridenMenuBehavior: Accessibility = (props: any) => {
+        const behavior = menuBehavior(props)
+      
+        behavior.focusZone.props.defaultTabbableElement = (root: HTMLElement): HTMLElement => {
+          return root.querySelector(".ui-menu__item__wrapper:last-child")
+        }
+      
+        return behavior
+      }
+      `}
+    />
+
+    <CodeSnippet label="App.jsx" value={`<Menu accessibility={overridenMenuBehavior} />`} />
+    <p>
+      All Stardust behaviors implementations can be found on the{' '}
+      {link(
+        'GitHub',
+        'https://github.com/stardust-ui/react/tree/master/packages/react/src/lib/accessibility/Behaviors',
+      )}
+      .
+    </p>
+    <Header as="h4" content="Available Behaviors" />
+    <p>
+      The default and other available behaviors for all the components can be found in the{' '}
+      {link('documentation', 'https://stardust-ui.github.io/react/')}, together with notes on other
+      accessibility considerations for using the component. The examples show the recommended way of
+      using the components in the different variations - it is possible to edit example's code, see
```

Comment:
""The examples show the recommended way of using the components in the different variations - it is possible to edit example's code, see the rendered HTML, change themes and validate the rendering in RTL scenario, or with different behaviors.""

maybe if you want to split the sentence somehow? :) it looks for me long, I haven't been able to read it until the end with understanding... just proposal :)